pkg/util/conntrack/conntrack_test.go

```diff
@@ -265,46 +235,42 @@ func TestDeleteConnections(t *testing.T) {
 	}
 }
 
-func TestClearConntrackForPortNAT(t *testing.T) {
+func TestClearUDPConntrackForPortNAT(t *testing.T) {
```

Comment:
I don't understand this change, we remove one test case of the existing 2, but you append new outputs as it there are 3 test cases, am I looking it wrong?

Reply:
fakeexec is just terrible... I tried to fix it once (https://github.com/kubernetes/utils/pull/14)

anyway, as it was, the PR correctly reverted the test back to how it was before 70020, but yes, it had the wrong number of entries before; the test case code checks that all of the expected outputs are correct, but it doesn't check that there are no extra expected outputs. Presumably someone copy+pasted this from one of the other test cases where there were more expected commands and forgot to delete the extras.

anyway, repushed with a fix